docs/source/etl/variant-data.rst

```diff
@@ -251,4 +256,9 @@ files must be located at ``{prefix}.bim`` and ``{prefix}.fam``.
 |                      |         |                 | If false, sets the sample ID to the individual ID.                                                  |
 +----------------------+---------+-----------------+-----------------------------------------------------------------------------------------------------+
 
+.. important::
+
+    The PLINK reader sets the first allele in the ``.bed`` file as the alternate allele, and the
+    second allele as an alternate allele.
```

Comment:
Looks like we missed this typo: Should be reference allele at the end of the sentence.